### DISCUSSION TITLE
Deluxe Menu Unranked and Ranked made by Shzv(the undersigned)

### DISCUSSION THREAD

**Discussion by Shzv** [diamond-rip/Eden#109] [General]
**REQUIRES DELUXE MENU**

for dowload: https://www.dropbox.com/s/clrnayqy00zm7l2/ranked-queue.yml?dl=1 and
https://www.dropbox.com/s/c30iis8nohtyfr6/unranked-queue.yml?dl=1

the unranked menu looks like this like mmc:
![2023-01-24_19 07 36](https://user-images.githubusercontent.com/122161612/214373317-388fb7f6-9bf1-43ed-902f-7826cf6beab0.png)
![2023-01-24_19 07 44](https://user-images.githubusercontent.com/122161612/214373343-392a7175-6fb9-4ac0-ba12-e74f9a92328b.png)

**Comment by GoodestEnglish** [comment]
Great to see this! In the coming update (might not be the next, but soon) user will be able to customize item's command. This will be great for feature like this.

**Comment by HierarchyMan** [comment]
Wow
